README.md

```diff
@@ -85,7 +66,7 @@ In order to run this project you need:
 Clone this repository to your desired folder:
 
 ```sh 
-git https://github.com/Salwa99/Ruby-school-library.git
+git clone https://github.com/Salwa99/Ruby-school-library.git
```

Comment:
**[OPTIONAL]**

- [ ] You have added an **HTTPS** link here but I strongly recommend that it would be great if you add an **SSH** link here because of authentication purposes :blush:

Reply:
Thank you @HAMMAS-SALEEM , I'll make sure to change it